extensions/amp-story/1.0/animation-presets-utils.js

```diff
@@ -48,7 +48,7 @@ export function translate2d(startX, startY, endX, endY) {
  */
 export function rotateAndTranslate(startX, startY, endX, endY, direction) {
   return [
-    {transform: translate(startX, startY) + ' ' + rotate(direction * 360)},
+    {transform: translate(startX, startY) + ' ' + rotate(direction * 120)},
```

Comment:
Is this a breaking change?

Reply:
Thanks for explaining during standup, I didn't understand it was the start value!